opal_hot_reloader.gemspec

```diff
@@ -23,14 +23,14 @@ Gem::Specification.new do |spec|
   end
 
   spec.files         = `git ls-files -z`.split("\x0").reject { |f| f.match(%r{^(test|spec|features)/}) }
-  spec.bindir        = "exe"
-  spec.executables   = spec.files.grep(%r{^exe/}) { |f| File.basename(f) }
+  spec.bindir        = "bin"
+  spec.executables   = spec.files.grep(%r{^bin/}) { |f| File.basename(f) }
   spec.require_paths = ["lib"]
 
   spec.add_development_dependency "bundler", "~> 1.10"
   spec.add_development_dependency "rake", "~> 10.0"
   spec.add_development_dependency "rspec"
 
-  spec.add_dependency 'listen', '3.1.1'
+  spec.add_dependency 'listen', '~> 3.0'
```

Comment:
Mine initially yelled at me for wanting 3.0.* to have the 'rerun' gem, but now that I look at it maybe ~> 3.1.0 would be just as good -- I think I needed to have 3.1.4, which is what it has now.

I looked at the changelog for listen, and 3.0 vs 3.1 doesn't look like it has any significant changes, and if I understand correctly '~> 3.0' will install the highest available 3.*.* version.